prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -21,6 +21,13 @@
 PARSER = argparse.ArgumentParser(
     description='Instrument speedtest.net speedtests from Prometheus.',
     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+PARSER.add_argument(
```

Comment:
Could you put the new output of **prometheus_speedtest --help** in the usage section of [README.md](https://github.com/jeanralphaviles/prometheus_speedtest/blob/050b234/README.md#Usage)?